pages/advanced/cli-demos.mdx

```diff
@@ -0,0 +1,15 @@
+# Demos
+
+## Create a new Safe
+
+The `PRIVATE_KEY` environment variable for this demo was set to an _EOA_ private key.
+
+[![Create a new Safe](https://asciinema.org/a/0jdHGLVRrkS9URxPoZ8ZJ7W2C.svg)](https://asciinema.org/a/0jdHGLVRrkS9URxPoZ8ZJ7W2C)
+
+## Sending Ether using a Trezor hardware wallet
```

Comment:
For consistency, the infinitive form should be used instead of the gerund (same in the link)